src/main/resources/application.yaml

```diff
@@ -65,11 +56,15 @@ app-insights:
   request-component: on
   telemetry-component: on
 
-rpa-dg-template-management-api:
-  base-url: ${DG_TEMPLATE_MANAGEMENT_API_URL:http://localhost:4623}
-
+# only used in functional tests
 idam:
   s2s-auth:
     url: ${S2S_BASE_URI:http://localhost:4502}
-    totp_secret: ${S2S_KEY:AAAAAAAAAAAAAAAA}
-    microservice: dg_template_management
+    totp_secret: ${FUNCTIONAL_TEST_CLIENT_S2S_TOKEN:AAAAAAAAAAAAAAAA}
+    microservice: em_gw
```

Comment:
Looking at the changes I'm not 100% sure whether this is used at runtime. If it is I will need to separate it from the functional test config